index.yaml

```diff
@@ -1750,15 +1750,15 @@ entries:
       repository: file://./charts/tracing
     description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
       for details.
-    digest: ef85e326a6dcc4b1d7f867f8e9cf43fe3ec82c9b08ca9458004fc60d06e37e3e
+    digest: 4774d1f117a1ce47230e2004dc23609f54ed49c6b45100b2438bf920822f7ca1
     icon: https://charts.rancher.io/assets/logos/istio.svg
     keywords:
     - networking
     - infrastructure
     name: rancher-istio
     urls:
-    - assets/rancher-istio-1.8/rancher-istio-1.8.601.tgz
-    version: 1.8.601
```

Comment:
This is another very frustrating diff to review but afai can tell this is the only thing actually getting removed. All the 1.9.x's stay as is and we are adding 1.10.2. Then for kiali we are swapping 1.32 for 1.35